src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependenciesTreeViewProvider.cs

```diff
@@ -383,7 +387,7 @@ IProjectTree UpdateTreeNode()
                 var updatedNodeParentContext = new ProjectTreeCustomizablePropertyContext
                 {
                     ExistsOnDisk = false,
-                    ParentNodeFlags = node.Parent?.Flags ?? default
+                    ParentNodeFlags = node!.Parent?.Flags ?? default
```

Comment:
Minor workaround of limitations in the static flow analysis. It does not deduce that this function is only called when `node` is not null.